README.md

```diff
@@ -278,6 +278,33 @@ exports.handler = signalFxLambda.asyncWrapper(async (event, context) => {
   tracer.scope().active().setTag("my-custom-tag", "some-value");
 });
 ```
+### Trace context propagation
+
+There's a built-in support for tracing across AWS Lambda boundaries when:
+- events are coming from Amazon API Gateway AWS Proxy 
+- OR events are executed programatically with a payload in which `headers` field contains B3-compatible key-value pairs, [see B3 spec for more details](https://github.com/openzipkin/b3-propagation)
+- AND outgoing requests are executed using HTTP (or HTTPS)
+
+No other scenarios are supported out-of-the-box (e.g. any requests coming from CloudFront or any Lambda destinations).
```

Comment:
If CloudFront is configured to allow forwarding tracing headers or all headers then it would work.

Reply:
Let me recheck this then. From what I've seen in examples, CF's data structure is different, ie. instead of `{headers: {...}}`, there's `{Records: [{headers: {...}}, ...]}`

Reply:
I see. Curious why CF event would have multiple records. Wouldn't a single call from CF represent a single HTTP call?

Reply:
Looking at https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-examples.html#lambda-examples-redirecting-examples, it should always be just one. I'm not sure why it's `{Records: []}` plural. I think we could support it, but I'd raise another ticket for it, because there's 4 different events with varying level of difficulty of implementation. There's also another subject of supporting propagation in SNS, SQS, and any other data streams, but I guess that should be left to be done manually by the user. DataDog do something with record ids but it's not exactly clear without a deeper look https://github.com/DataDog/datadog-lambda-js/blob/b2de6899d5b7d01e592c4a773013f874ca07c7ca/integration_tests/process-input-traced.js#L47

Reply:
Sounds good.